mfr/extensions/ipynb/templates/viewer.mako

```diff
@@ -8,8 +8,7 @@
 </div>
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js"></script>
```

Comment:
Mathjax can't easily be webpacked yet, so leaving this on their cdn for now is fine. the script inlined below can be webpacked though.